Assets/_BForBoss/_Analytics/Scripts/PerigonAnalytics.cs

```diff
@@ -1,6 +1,7 @@
 using System;
 using System.Collections;
 using mixpanel;
+using UnityEngine;
```

Comment:
Is this needed?

Reply:
I needed this in order to use the device unique identifier. If we don't want the UnityEngine import, I can try looking for another unique ID

Reply:
technically you can inject the string instead. So that it's not couple to unity engine

Reply:
I'll let you decide

Reply:
yup that's probably better. I made the changes